expo/expo
# [ios][sdk36] Add React-RCTCameraRoll to SDK36 codebase

# Why

Part of iOS Quality Assurance process (#6264). CameraRoll is no longer included within React Native which causes that there is no appropriate request handler/image loader to load images with `assets-library://` url scheme which come from user's media library. 

# How

Added temporary `ABI36_0_0React-RCTCameraRoll.podspec` to the codebase versioned for SDK36 and installed this pod.

*I've decided not to fix this issue in `UNVERSIONED` environment as we're planning to come up with a better solution for this problem in the next SDK.*

# Test Plan

- Set `sdkVersion` in `apps/native-component-list/app.json` to `36.0.0`.
- Start `native-component-list` and open up `MediaLibrary` example - it should show photos from your user's library.
